Examples/hello-world-hummingbird-server-otlp-http-json/docker/otel-collector-config.yaml

```diff
@@ -0,0 +1,36 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: "otel-collector:4317"
```

Comment:
Same comment as above. Do we want to update the Docker files in the examples to only expose the ports/endpoints that we need for the example to run?

Reply:
Yeah, also fine by me.